src/bin/jcli_app/block/genesis/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -38,8 +38,9 @@ blockchain_configuration:
   # Genesis praos parameter D
   consensus_genesis_praos_param_d: 0.220
 
-  # Genesis praos parameter F
-  consensus_genesis_praos_param_f: 0.220
+  # Genesis praos active slot coefficient
+  # Determines minimum stake required to try becoming slot leader, must be in range (0,1]
+  consensus_genesis_praos_active_slot_coeff: 0.22
```

Comment:
Good as the immediate fix.

One observation though. YAML is a structured format. Why do we have mile-long field names instead of:

```yaml
consensus:
  genesis_praos:
    param_d: 0.220
    active_slot_coeff: 0.22    
```

Reply:
Filed the above as #324

Reply:
Nice!